Mathlib/Analysis/Analytic/IsolatedZeros.lean

```diff
@@ -240,6 +240,44 @@ lemma apply_eq_zero_of_order_toNat_ne_zero (hf : AnalyticAt 𝕜 f z₀) :
   simp [hf.order_eq_zero_iff]
   tauto
 
+/- Helper lemma for `AnalyticAt.order_mul` -/
+lemma order_eq_top_of_order_eq_top_mul_analytic {f g : 𝕜 → 𝕜} (hf : AnalyticAt 𝕜 f z₀)
```

Comment:
This name doesn't entirely make sense to me. How about
```suggestion
lemma order_mul_eq_top_of_order_eq_top {f g : 𝕜 → 𝕜} (hf : AnalyticAt 𝕜 f z₀)
```
or even
```suggestion
lemma order_mul_of_order_eq_top {f g : 𝕜 → 𝕜} (hf : AnalyticAt 𝕜 f z₀)
```

Reply:
The second one is much better (to me), thanks!